scientific/ne512_Test_Case/ne512_TMQ_v2_Test_Case.ipynb

```diff
@@ -0,0 +1,301 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f=cdms2.open(\"./add-ne512-grids.FC5AV1C-H01A.ne512np4_360x720cru_ne512np4.cori-knl_intel.1024x17x8.ifs-hindcast.20190610-1542.cam.h1.2016-08-01-00000.360x720cru_3D_all_hind.nc\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initialize the VCS canvas\n",
+    "In this example, the VCS canvas is called \"x\"."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = vcs.init()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set the DV3D settings\n",
+    "Choose various DV3D settings for the type of plot you want to make."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dv3d = vcs.get3d_scalar()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dv3d.ToggleVolumePlot = vcs.on"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dv3d.Camera={'Position': (-161, -171, 279), 'ViewUp': (.29, 0.67, 0.68), 'FocalPoint': (146.7, 8.5, -28.6)}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load TMQ data\n",
+    "\n",
+    "The variable \"v\" holds the 3D TMQ (total (vertically integrated) precipitable water) test data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "v = f[\"TMQ\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next line determines the min and max of values for the \"v\" variable in our NetCDF file. It might take a while for this line to run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import genutil\n",
+    "genutil.minmax(v)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Modify the ScaleColormap to cover the min and max values of the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dv3d.VerticalScaling = 4.0 \n",
+    "dv3d.ScaleColormap = [0.07, 169.0, 1] \n",
+    "dv3d.ScaleTransferFunction =  [0.07, 169.0, 1]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "NOTE: when running the Python script outside of the Jupyter notebook in order to interact with the data visualization, I received the following error when I tried to adjust the \"Range Max\" slider which controls the maximum data value that is shown on the 3D plot:\n",
```

Comment:
Did you want to leave that note for the user? Or is that for developer?

Reply:
For this notebook, the users are essentially developers since the audience is the E3SM project developers, not the general public.  This notebook is NOT intended to be released to the general public, mainly because the data should remain within the E3SM project.